src/data/multiset/basic.lean

```diff
@@ -2135,6 +2135,13 @@ def subsingleton_equiv [subsingleton α] : list α ≃ multiset α :=
   left_inv := λ l, rfl,
   right_inv := λ m, quot.induction_on m $ λ l, rfl }
 
+variable {α}
+
+@[simp]
+lemma subsingleton_equiv_apply' [subsingleton α] :
```

Comment:
```suggestion
lemma subsingleton_equiv_coe [subsingleton α] :
```
or maybe
```suggestion
lemma coe_subsingleton_equiv [subsingleton α] :
```
since there's no application of the equiv going on here